app/scripts/lib/auth-errors.js

```diff
@@ -191,6 +191,14 @@ function (_, Errors) {
     FORCE_AUTH_EMAIL_REQUIRED: {
       errno: 1024,
       message: t('/force_auth requires an email')
+    },
+    EXPIRED_VERIFICATION_LINK: {
+      errno: 1025,
+      message: t('The link you clicked to verify your email is expired.')
```

Comment:
The string comes from the [complete_sign_up template](https://github.com/mozilla/fxa-content-server/blob/02e62fb54e033dbec5cec9ef6881e5987b9d9f33/app/scripts/templates/complete_sign_up.mustache#L12). I kept the `.` so localizers would not need to translate two subtly different strings.